pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_comparison-between-modules-and-topics.adoc

```diff
@@ -0,0 +1,13 @@
+include::../attributes.adoc[]
+[id='comparison-between-modules-and-topics_{context}']
+= Comparison between modules and topics in traditional topic-based authoring
+
+A module, in {RH}'s modular documentation approach, is similar to a topic in traditional topic-based authoring. The differences between the two are in scope, and application - whereas a topic is a general and abstract concept, a module is a concrete implementation of the abstract concept specific to modular documentation, similar to how a DITA topic defined by the OASIS group is a specific implementation of a topic.
+
+For a topic-based content to be a module, the content must conform to the definition a module as described in Modules. 
```

Comment:
Rewriting to remove some of the repetition and the (in my opinion, unnecessary) comma. Is this still accurate?

"To be considered a module, topic-based content must conform to the definition of a module."